### PR TITLE
fix: map links from fish guide

### PR DESCRIPTION
**Pull request by TobiTenno**
**Summary:**
fixes an issue yazu reported in discord

image

---
**Fixes issue (include link):**

---
**Mockups, screenshots, evidence:**

### CODE REVIEW

**Review by EricSihaoLin — COMMENTED** [review]
Would it be possible to fix the hotspot link too?

**Comment by TobiTenno** [comment]
> Would it be possible to fix the hotspot link too?

for sure!

uhhh, actually, @EricSihaoLin which is broken?

File: pages/ow/fish/howto/index.vue [hunk 1]
```diff
@@ -270,7 +270,7 @@
               an insane amount of standing and resources that you will need for later on. To do so, you need the
               appropriate bait to maximize the chances for those fish to spawn. Rare fishes do spawn in Orb Vallis
               without their respective bait, but definitely not as often compared to when you do use their bait. Again,
-              you can refer to the <nuxt-link to="vallis-fish">Super Fishing Reference Table</nuxt-link> for its
+              you can refer to the <nuxt-link to="vallis/fish">Super Fishing Reference Table</nuxt-link> for its
```
Comment (EricSihaoLin) [inline pages/ow/fish/howto/index.vue]: For some reason this and all other links I have tested still redirects to `ow/fish/poe-fish` instead of `poe/fish`

Comment (TobiTenno) [reply pages/ow/fish/howto/index.vue]: might need to purge locally :thinking: i'll try out the vercel deploy

Comment (TobiTenno) [reply pages/ow/fish/howto/index.vue]: oh, looks like there's like 4 of the exact same link

Comment (TobiTenno) [reply pages/ow/fish/howto/index.vue]: @EricSihaoLin it's looking right for me now if you open the latest deployment in an incognito tab

**Comment by TobiTenno** [comment]
@EricSihaoLin i rebased, could you try again?

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**